src/Sulu/Bundle/ContactBundle/DependencyInjection/SuluContactExtension.php

```diff
@@ -100,6 +100,19 @@ public function prepend(ContainerBuilder $container)
                                     ],
                                 ],
                             ],
+                            'single_contact_selection' => [
```

Comment:
This should already have been added... Can you rebase?

Reply:
rebased!